subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfiguration.java

```diff
@@ -481,7 +481,7 @@ public void run(BuildOperationContext context) {
 
             @Override
             public BuildOperationDescriptor.Builder description() {
-                return BuildOperationDescriptor.displayName("Resolve dependencies of " + identityPath).progressDisplayName("Resolve dependencies " + identityPath);
+                return BuildOperationDescriptor.displayName("Resolve dependencies of " + identityPath).progressDisplayName("Resolving dependencies " + identityPath);
```

Comment:
This might have bad side effects on some cross-version TAPI tests like:
https://github.com/gradle/gradle/blob/e1e0bba09aff69f2de540426124852077fe9e9d7/subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/r33/BuildProgressCrossVersionSpec.groovy#L206-L206